main/src/ca/uwaterloo/flix/language/phase/LoadBytecode.scala

```diff
@@ -57,11 +57,14 @@ object LoadBytecode extends Phase[ExecutableAst.Root, ExecutableAst.Root] {
     * 5. Load Enum classes
```

Comment:
I think this entire class can be replaced by two for loops. One for loop over a very general structure to load every generated interface and class. And one for loop over each definition symbol to load its associated method (via a helper function that maps a definition symbol to a java name),

Reply:
We should talk about this and add some tickets for it!